App.tsx

```
@@ -3,6 +3,7 @@ import Toast from 'react-native-toast-message';
 import SplashScreen from 'react-native-splash-screen';
 import { NavigationContainer } from '@react-navigation/native';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
+import analytics from '@react-native-firebase/analytics';
```

ESLint

[error] 'analytics' is defined but never used.

(@typescript-eslint/no-unused-vars)